src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/GeneralFeedback/messages.js

```diff
@@ -0,0 +1,23 @@
+export const messages = {
+  generalFeebackSettingTitle: {
+    id: 'authoring.problemeditor.settings.generalFeebackSettingTitle',
+    defaultMessage: 'General Feedback',
+    description: 'label for general feedback setting',
+  },
+  generalFeedbackInputLabel: {
+    id: 'authoring.problemeditor.settings.generalFeedbackInputLabel',
+    defaultMessage: 'Enter the feedback to appear when a student submits a wrong answer. This will be overrdidden if you add answer-specific feedback.',
```

Comment:
Shouldn't the defaultMessage be "Feedback message"

Reply:
spelling nit: overridden